README.markdown

```diff
@@ -1037,198 +1431,252 @@ The second argument to this function is which type of hash to use. It will be co
+*Note:* This function is an implementation of a Ruby class and might not be UTF8 compatible. To ensure compatibility, use this function with Ruby 2.4.0 or greater.
 
 #### `strftime`
 
-Returns formatted time. For example, `strftime("%s")` returns the time since Unix epoch, and `strftime("%Y-%m-%d")` returns the date. *Type*: rvalue.
-
-*Please note:* This function is an implementation of a Ruby class and as such may not be entirely UTF8 compatible. To ensure compatibility please use this function with Ruby 2.4.0 or greater - https://bugs.ruby-lang.org/issues/10085.
-
-  *Format:*
-
-    * `%a`: The abbreviated weekday name ('Sun')
-    * `%A`: The full weekday name ('Sunday')
-    * `%b`: The abbreviated month name ('Jan')
-    * `%B`: The full month name ('January')
-    * `%c`: The preferred local date and time representation
-    * `%C`: Century (20 in 2009)
-    * `%d`: Day of the month (01..31)
-    * `%D`: Date (%m/%d/%y)
-    * `%e`: Day of the month, blank-padded ( 1..31)
-    * `%F`: Equivalent to %Y-%m-%d (the ISO 8601 date format)
-    * `%h`: Equivalent to %b
-    * `%H`: Hour of the day, 24-hour clock (00..23)
-    * `%I`: Hour of the day, 12-hour clock (01..12)
-    * `%j`: Day of the year (001..366)
-    * `%k`: Hour, 24-hour clock, blank-padded ( 0..23)
-    * `%l`: Hour, 12-hour clock, blank-padded ( 0..12)
-    * `%L`: Millisecond of the second (000..999)
-    * `%m`: Month of the year (01..12)
-    * `%M`: Minute of the hour (00..59)
-    * `%n`: Newline (\n)
-    * `%N`: Fractional seconds digits, default is 9 digits (nanosecond)
-      * `%3N`: Millisecond (3 digits)
-      * `%6N`: Microsecond (6 digits)
-      * `%9N`: Nanosecond (9 digits)
-    * `%p`: Meridian indicator ('AM' or 'PM')
-    * `%P`: Meridian indicator ('am' or 'pm')
-    * `%r`: Time, 12-hour (same as %I:%M:%S %p)
-    * `%R`: Time, 24-hour (%H:%M)
-    * `%s`: Number of seconds since the Unix epoch, 1970-01-01 00:00:00 UTC.
-    * `%S`: Second of the minute (00..60)
-    * `%t`: Tab character (	)
-    * `%T`: Time, 24-hour (%H:%M:%S)
-    * `%u`: Day of the week as a decimal, Monday being 1. (1..7)
-    * `%U`: Week number of the current year, starting with the first Sunday as the first day of the first week (00..53)
-    * `%v`: VMS date (%e-%b-%Y)
-    * `%V`: Week number of year according to ISO 8601 (01..53)
-    * `%W`: Week number of the current year, starting with the first Monday as the first day of the first week (00..53)
-    * `%w`: Day of the week (Sunday is 0, 0..6)
-    * `%x`: Preferred representation for the date alone, no time
-    * `%X`: Preferred representation for the time alone, no date
-    * `%y`: Year without a century (00..99)
-    * `%Y`: Year with century
-    * `%z`: Time zone as hour offset from UTC (e.g. +0900)
-    * `%Z`: Time zone name
-    * `%%`: Literal '%' character
+Returns formatted time.
+
+For example, `strftime("%s")` returns the time since Unix epoch, and `strftime("%Y-%m-%d")` returns the date.
```

Comment:
link to upstream documentation for full details: https://ruby-doc.org/core-2.1.9/Time.html#method-i-strftime